backend/src/api/model/block/mutations.rs

```diff
@@ -300,6 +300,38 @@ impl BlockValue {
         Ok(Self::from_row(updated_block)?)
     }
 
+    pub(crate) async fn update_video(
+        id: Id,
+        set: UpdateVideoBlock,
+        context: &Context
```

Comment:
Trailing comma

![](https://i.pinimg.com/originals/d2/e4/e4/d2e4e412376f33edfd657d3373f8c245.gif)

Reply:
It's gonna be fine.